lib/evm/selectors/index.js

```diff
@@ -141,7 +141,32 @@ const evm = createSelectorTree({
     /**
      * evm.info.binaries
      */
-    binaries: createLeaf(['/state'], (state) => state.info.contexts.byBinary)
+    binaries: {
+      _: createLeaf(['/state'], (state) => state.info.contexts.byBinary),
+
+      /**
+       * evm.info.binaries.search
+       *
+       * returns function (binary) => context
+       */
+      search: createLeaf(['./_'], (binaries) => {
+        // HACK ignore link references for search
+        // link references come in two forms: with underscores or all zeroes
+        const toRegExp = (binary) =>
```

Comment:
Just out of curiosity - who/what generates the two forms?

Reply:
Added a comment about this. Thanks!